tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Steps/ServiceSearchSteps.cs

```diff
@@ -16,64 +16,86 @@ namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Steps;
 /// </summary>
 public class ServiceSearchSteps
 {
-    readonly ConfigModel config;    
-    private readonly string baseUrl;
-    private ServiceSearchRequest request;
-    private HttpResponseMessage lastResponse;
-    private HttpStatusCode statusCode;
-    private const string serviceSearchPath = "api/metrics/service-search";
-    public ServiceSearchSteps()
-    {
-        config = ConfigAccessor.GetApplicationConfiguration();
-        baseUrl = config.BaseUrl;
-    }
+  readonly ConfigModel config;  
+  private readonly string baseUrl;
+  private ServiceSearchRequest request;
+  private HttpResponseMessage lastResponse;
+  private HttpStatusCode statusCode;
+  private const string serviceSearchPath = "api/metrics/service-search";
+  public ServiceSearchSteps()
+  {
+      config = ConfigAccessor.GetApplicationConfiguration();
+      baseUrl = config.BaseUrl;
+  }
 
-    #region Step Definitions
+  #region Step Definitions
 
-    #region Given
+  #region Given
+ 
 
-    public void GivenIHaveAPostcodeToSearchWithinFind()
-    {
-        DateTime time = DateTime.UtcNow;
+  public void GivenIPostToTheServiceSearchEndpointWithParameters(string radiusValue, string postcodeEntry, string postCodeEndpointResponseEntry,string searchTriggerEventId,string serviceSearchTypeId)
+  {
+      DateTime time = DateTime.UtcNow;
+      var radius = int.Parse(radiusValue);
+      string postcode = postcodeEntry;
+      var postcodeEndpointStatusCode = int.Parse(postCodeEndpointResponseEntry);
+      var searchTriggerEventIdEntry = int.Parse(searchTriggerEventId);
+      var serviceSearchTypeIdEntry = int.Parse(serviceSearchTypeId);
```

Comment:
refactor: looks like we're mixing usages of `var` and explicit types here. Let's stick to using explicit types if possible :)

`var radius = int.Parse(radiusValue);` -> `int radius = int.Parse(radiusValue);`